packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1841,7 +1850,19 @@ class _JsonViewerState extends State<JsonViewer>
     );
     // Intended to be unawaited.
     // ignore: discarded_futures
-    _initializeTree = buildVariablesTree(variable);
+    _initializeTree = _buildAndExpand(variable);
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    _updateVariablesTree();
+  }
+
+  @override
+  void didUpdateWidget(covariant JsonViewer oldWidget) {
```

Comment:
Why is this covariant?

Reply:
Looks like maybe the autocomplete picked it up from inside state.